.github/CODEOWNERS

```diff
@@ -432,7 +430,7 @@
 /main_cron.py @nithusha21
 /main_mail.py @aks681
 /main.py @DubeySandeep @nithusha21
-/feconf.py @seanlip @nithusha21
+/feconf.py @DubeySandeep @nithusha21 @seanlip
```

Comment:
Considering you are going on vacation, do you want to be assigned to this file?

Reply:
I think in this case it's OK since there are backup codeowners. But I'll change the other instances since I forgot there's going to be a release soon (I had originally left myself assigned since I figured that there's not going to be changes to AUTHORS etc. while I was away). Thanks for pointing it out!

Reply:
@seanlip in general do you think it's okay for code owners to be a part of the pattern (while being on vacation) if there are multiple code owners in that section. (Asking this as @sajalasati is writing a design doc for a script that will help code owners to do temporary code ownership transfer.) 

/cc @sajalasati,

Reply:
Confirming that what @sajalasati is doing (and what you are suggesting) is correct. I'm actually only doing it this way to reduce the number of back-changes I need to do afterwards, but a script would solve a lot of these problems so I'm very glad to hear that it's being worked on! (Thanks @sajalasati :) )